tests/B4RealTest.spec.ts

```diff
@@ -7,17 +7,21 @@ import { getRevertMessage } from "./utils";
 
```

Comment:
If you use the `ethereum-waffle` middleware you can handle BigNumbers in your assertions.

```suggestion
import { solidity } from "ethereum-waffle";

chai.use(solidity);
```

Then you can go 
```
expect(myBigNumber).to.equal(anotherBigNumber);
```